### PR TITLE
Issue191 fuse updates

### PR DESCRIPTION
**Pull request by bredamc**
Addresses #191

### CODE REVIEW

**Comment by bredamc** [comment]
@bergerhoffer I've implemented the changes requested by @MelissaFlinn now, and asked Melissa to check to make sure I've updated correctly.

I wasn't sure whether or not to continue your practice of inserting comments to describe the changes? Because it's easier to remember the changes while I'm making them, I included the comments for now (tagged with my name to distinguish them from yours), but I can remove them if you prefer :)

**Review by MelissaFlinn — APPROVED** [review]

**Review by MelissaFlinn — DISMISSED** [review]

**Review by bergerhoffer — DISMISSED** [review]
@bredamc Thanks, this looks great to me, nice job!

As for the comments - no it's not something we need to keep doing. I just did it because when I moved things around, it looked like all brand new content, so I added comments to point out if I did make any changes to it when moving. I might go in and clean those up some day, but it's not a high priority. So definitely not hurting anything to keep your comments in! Thanks again!

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council ready for some more reviews please!

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc:189]: For 1) and 2) _node_ should be in italics :), as it's explains a node.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:354]:
```suggestion
*Description*: The distribution of Red Hat Fuse for non-expert integrators with a simplified workflow that is accessed through a browser-based UI.
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:981]: Consider using bullet points to avoid the passives:

Routing rules provide the following tasks:

- Specify the source and type of input.
- Determine the mechanism for processing any input.
- Direct any output to a source.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:1019]: IBM SG tends to reserve parentheses for unique cases, such as abbreviations. Consider:

```suggestion
*Description*: In Red Hat Fuse, routing rules are declarative statements, which are written in Java or XML DSL, that define the paths which messages must take from their origin, or source, to their target destination, which is also known as a _sink_. Routing rules start with a consumer endpoint, `from`, and typically end with one or more producer endpoints, `to`,. Between consumer and producer endpoints, messages can enter various processors, which might transform them or redirect them to other processors or to specific producer endpoints.
```

**Comment by dfitzmau** [comment]
Content looks well. I added some small suggestions.

**Comment by bergerhoffer** [comment]
@bredamc Can you PTAL at @dfitzmau's suggestions when you get a chance?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc:189]: @dfitzmau I'm so sorry for the delay in replying to you -- thanks so much for taking the time to review and send feedback!

I'll reply to each comment individually -- I agree with this one and will make the change. I'll also rephrase 3 slightly to italicise that one too.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc:208]: @MelissaFlinn You're the SME here! Can we use a hyphen in this case ("attribute-value pairs" instead of "attribute/value pairs")?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:354]: Agreed

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:981]: Sorry @dfitzmau -- I actually think the original text is better than the suggested rewrite in this case! 
@MelissaFlinn @bergerhoffer, what do you think?

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (bredamc) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:1019]: I take your point and will rewrite in a way that I hope satisfies us both :)

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:981]: We actually usually try to avoid doing complex descriptions (I know we have some, but :shrug:). There are some tools out there that parse our glossary entries (Vale for example I think is, or is trying to). So I'd recommend keeping how it is now too.

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc:981]: NP. I'll mark this comment as resolved. Thanks all.

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc
Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc:189]: Hi @bredamc . Thanks for making the changes. Node should be surrounded by italics as these sentences are describing _node_ and not stating how one should use the term "node". PRs are in progress that are globally implementing these changes to the RH SSG. 

Would you be OK to change your text inline with the italics and double quotes pending guidelines?

**Comment by bredamc** [comment]
@dfitzmau Finally implemented your suggestions -- please let me know what you think, and thanks again for taking the time to review and send your comments.

@MelissaFlinn Please also review the latest changes, to ensure that I haven't inadvertently introduced any errors. You're more familiar with the subject matter than I am :)

**Review by dfitzmau — APPROVED** [review]

**Comment by dfitzmau** [comment]
> @dfitzmau Finally implemented your suggestions -- please let me know what you think, and thanks again for taking the time to review and send your comments.
> 
> @MelissaFlinn Please also review the latest changes, to ensure that I haven't inadvertently introduced any errors. You're more familiar with the subject matter than I am :)

Hi @bredamc . NP. I know what it is like. Thanks for reviewing my comments and implementing some changes. I approved the PR. Nice work!

**Review by ndeevy — APPROVED** [review]

**Comment by ndeevy** [comment]
This PR looks good to me.

**Review by MelissaFlinn — APPROVED** [review]
LGTM

**Review by ndeevy — APPROVED** [review]

**Comment by bredamc** [comment]
Oops, didn't mean to dismiss existing approval, sorry @dfitzmau -- could you please approve again?
@bergerhoffer Can you please approve again too?

Thanks @MelissaFlinn for reviewing and approving :)

**Review by dfitzmau — APPROVED** [review]

**Comment by jseseCCS** [comment]
LGTM too, if we still need another review. (saw Nicole's ping in the G channel.)

**Review by bburt-rh — APPROVED** [review]

**Pull request merged**
